src/Esfa.Roaao.Xslx.IntegrationTests/RoaaoExcelTests.cs

```diff
@@ -91,6 +91,17 @@ public void EpaShouldExist()
 
         [TestMethod]
         [TestCategory("RoAAo Excel Tests")]
+        public void ShouldNotHaveSpaceInOrganisationType()
```

Comment:
Naming can be improved, "Training Provider" is a valid organisation type and contains a space into it